common/reviews/api/tools.api.md

```diff
@@ -105,7 +105,7 @@ export class AngleTool extends AnnotationTool {
     // (undocumented)
     _activateModify: (element: HTMLDivElement) => void;
     // (undocumented)
-    addNewAnnotation: (evt: EventTypes_2.MouseDownActivateEventType) => AngleAnnotation;
+    addNewAnnotation: (evt: EventTypes_2.MouseDownActivateEventType | EventTypes_2.TouchStartActivateEventType) => AngleAnnotation;

```

Comment:
These changes make the 95% case hard, which is that mouse and touch handling are identical.  It also makes adding other custom types hard as well, because they need changes everywhere to handle the default case, which is "mouse like" events.  What you want to do is make everything handle it as though it was a mouse event, EXCEPT:
1. If the tool needs special handling that is different for a specific controller (eg touch controller)
2. Registering subsequent event handlers need to register the right type

For the second item, I mean if you touch down, then you do NOT want to listen to "mouse move", if you add a "mouse move" listener, but what you want is to listen to the move listener appropriate to the touch event.